backend/src/db/repositories/UserRepository.ts

```diff
@@ -37,6 +38,18 @@ export default class UserRepository {
         });
     }
 
+    async getFeedSortingSettingsByUserId(userId: number): Promise<FeedSortingSettingsRaw[] | undefined> {
+        return await this.db.query<FeedSortingSettingsRaw[]>('select * from feed_sorting_settings where user_id=:user_id', {user_id: userId});
+    }
+
+    async saveFeedSorting(siteId: number, feedSorting: FeedSorting, userId: number) {
+        await this.db.query(`replace into feed_sorting_settings (user_id, site_id, feed_sorting) values (:user_id, :site_id, :feed_sorting)`, {
```

Comment:
It's not that important here, just advice.
I personally prefer `insert ... on duplicate key update ...`.
`replace into` removes existing record instead of updating existing one, that is a bit slower.